src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -87,7 +89,7 @@ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.Rea
       comments={item?.rawJson?.comments?.length}
       downvote={item?.rawJson?.downvote}
       isCommentExists={Boolean(firstComment)}
-      isMe={item?.user?.isMe || isOwnSignedPost || isOwnPost}
+      isMe={getIsMe()}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The replacement of the `isMe` prop value with the `getIsMe()` function call seems to be a good change as it makes the code more readable and maintainable. However, please ensure that the `getIsMe()` function correctly implements the logic that was previously used to determine the value of `isMe`.
```diff
-      isMe={item?.user?.isMe || isOwnSignedPost || isOwnPost}
+      isMe={getIsMe()}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->